source/tutorials_installation.md

```diff
@@ -0,0 +1,396 @@
+  connection:
+    enabled: true
+    decrypt_token: somenotemptystring
+    repository_type: kubernetes
+  deployment:
+    edge:
+      host: http://odahu.example.com
+EOF
+```
+
+
+**Note**:
+> This example uses hostname `odahu.example.com` as entrypoint for cluster services.
+>
+> It should point to LoadBalancer IP from [ingress section](tutorials_installation.html#install-nginx-ingress).
+
+In order to setup ODAHU services along with ready-to-use [connections](ref_connections.html), you may add according section to values YAML in advance:
+
+```
+connections:
+- id: docker-hub
+  spec:
+    description: Docker registry for model packaging
+    username: "user"
+    password: "supersecure"
+    type: docker
+    uri: docker.io/odahu-models-repo
+    webUILink: https://hub.docker.com/r/odahu-models-repo
+```
+
+[Read more about connections](ref_connections.html).
+
+
+Install odahu-flow core services:
+```bash
+$ helm install odahu/odahu-flow-core --name odahu-flow --namespace odahu-flow --values ./odahuflow_values.yaml
+```
+
+### Training service (MLFlow)
+
+Prepare YAML config with values for [`odahu-flow-mlflow`](https://github.com/odahu/odahu-trainer/tree/develop/helms/odahu-flow-mlflow) Helm chart.
+
+```bash
+$ cat << EOF > mlflow_values.yaml
+logLevel: debug
+ingress:
+  globalDomain: example.com
+  enabled: true
+tracking_server:
+  annotations:
+    sidecar.istio.io/inject: "false"
+toolchain_integration:
+  enabled: true
+EOF
+```
+
+Install Helm chart:
+
+```bash
+$ helm install odahu/odahu-flow-mlflow --name odahu-flow-mlflow --namespace odahu-flow \
+    --values ./mlflow_values.yaml
+```
+
+### Packaging service
+
+Install [`odahu-flow-packagers`](https://github.com/odahu/odahu-packager/tree/develop/helms/odahu-flow-packagers) Helm chart:
+
+```bash
+$ helm install odahu/odahu-flow-packagers --name odahu-flow-packagers --namespace odahu-flow
+```
+
+### Delete ODAHU services
+
+To delete and purge Helm chart run:
+```bash
+$ helm delete --purge odahu-flow
+```
+
+To clean up remaining `CustomResourceDefinitions` execute following command:
+```bash
+$ kubectl get crd | awk '/odahuflow/ {print $1}' | xargs -n1 kubectl delete crd
+```
+
+To purge everything installed in previous steps with single command, run
+```bash
+$ helm delete --purge odahu-flow-packagers odahu-flow-mlflow odahu-flow &&\
+  kubectl delete namespace odahu-flow &&\
+  for i in training packaging deployment; do \
+    kubectl delete namespace odahu-flow-${i} || true; done &&\
+  kubectl get crd | awk '/odahuflow/ {print $1}' | xargs -n1 kubectl delete crd &&\
+  kubectl -n istio-system delete job.batch/odahu-flow-feedback-rq-catcher-patcher &&\
+  kubectl -n istio-system delete sa/odahu-flow-feedback-rq-catcher-patcher &&\
+  kubectl -n istio-system delete cm/odahu-flow-feedback-rq-catcher-patch
+```
+
+## Install additional services
+
+### Install [Prometheus operator](https://github.com/coreos/prometheus-operator) (optional)
+
+TBD
+
+## Conclusion
```

Comment:
Are you going to finish this section in this PR?

Reply:
I decided to leave only the necessary services, and optional ones just list with links to charts.
Done in 371ff96ef72f55bc19400437cf84f8b976148657.